### PR TITLE
Timeline slider

### PR DESCRIPTION
**Pull request by EB25Ball**
This should work. Let me know if something errors.

### CODE REVIEW

**Comment by Vestria4** [comment]
🦁

File: website/levels/viewer/scripts/level_viewer.js [hunk 1]
```diff
@@ -860,21 +864,31 @@ function onWindowResize()
...
+			animationTime=parseInt(document.getElementById('time-slider').value)
+		}
+	else {
+			const delta = clock.getDelta();
+			controls.update(delta);
+			animationTime += delta;
+			document.getElementById('time-slider').value = animationTime
+			for (let object of animatedObjects) {
+				updateObjectAnimation(object, animationTime)
+			}
```
Comment (Slin) [inline website/levels/viewer/scripts/level_viewer.js:888]: Shouldn't this be outside the else?

Comment (EB25Ball) [reply website/levels/viewer/scripts/level_viewer.js]: 👍

**Comment by twhlynch** [comment]
@EB25Ball why is it spamming the console -.-

**Pull request merged**